### ISSUE TITLE
Login: Redirected to the /login screen from 2FA screens

### ISSUE THREAD

**Issue by scruffian** [Automattic/wp-calypso#14104]
When clicking around inside the 2FA screens in login I send up getting a `invalid_two_step_nonce` error and being redirected to the main login screen:

<img width="521" alt="screen shot 2017-05-16 at 16 25 04" src="https://cloud.githubusercontent.com/assets/275961/26114106/588bd8ee-3a54-11e7-8188-d6d45f8ba737.png">

Steps to reproduce:
1. Go to /login
2. Enter a username and password for account with 2FA enabled
3. Click on "The Authenticator App"
4. Click on "The WordPress mobile app"
5. Repeat steps 3 and 4 for a while...
4. After a while you'll be redirected back to the login form

**Comment by fabianapsimoes** [comment]
I don't get this error notice, though I am redirected back to the login form when I follow these steps.

I'd call this low priority, since it's fairly edgy behavior to randomly click links like this. I actually think it's an OK behavior to redirect to login, as this kind of interaction can look quite spammy / bot-like.

Do you have steps to reproduce that exact error message? Maybe we can replace it with something like "Having trouble logging in? Get help". But not sure I'd go that far if it's a tricky thing to do.

**Comment by drewblaisdell** [comment]
I can't reproduce this after #14539.
